import-service/src/helper.js

```diff
@@ -0,0 +1,23 @@
+const headers = {
+    'Access-Control-Allow-Origin': '*',
+    'Access-Control-Allow-Credentials': true,
+};
+
+const response = (body, statusCode) => {
+    body = JSON.stringify(body);
+    return {
+        statusCode,
+        body,
+        headers
+    };
+}
+
+module.exports.successResponse = body => {
+    return response(body, 200);
```

Comment:
200, 400, 500 - could be enum constant and could be used everywhere

Reply:
@DrDiman Good point, done